File: AlphaArray.cpp
#include <iostream>
using namespace std;
int main() {
   
   char alpha[26];
    
    for(int i =0;i<=26;i++){
       alpha[i]=i +'a';
    }
    
    for(int i=0;i<26;i++){
        cout<<alpha[i]<<endl;
    }


    return 0;
}

File: AlphaArray.java
class alpha {
    public static void main(String[] args) {
        char[] alpha = new char[26];
        for(int i = 0; i < 26; i++) {
            alpha[i] = (char) (i + 'a');
        }
        
        for(int i = 0; i < 26; i++) {
            System.out.println(alpha[i]);
        }
    }
}
